### PR TITLE
airflow: marquez integration wraps openlineage-airflow, remove independent implementation

### PR DESCRIPTION
**Pull request by mobuchowski**
In preparation for the Marquez 0.17.0 release, make `marquez-airflow` just wrap `openlineage-airflow` release. `marquez-airflow` is deprecated and will be deleted in some following release.

Signed-off-by: Maciej Obuchowski <maciej.obuchowski@getindata.com>

### CODE REVIEW

**Review by wslulciuc — CHANGES_REQUESTED** [review]
We'll still want to keep the integration test to ensure the Airflow wrapper still works with the Marquez backend. Mind add the [integration test](https://github.com/MarquezProject/marquez/tree/main/integrations/airflow/tests/integration) and the `integration-test-airflow` CI job that runs the integration test?

File: integrations/airflow/README.md [hunk 1]
```diff
@@ -1,154 +1,9 @@
-# marquez-airflow
+# DEPRECATED
```
Comment (wslulciuc) [inline integrations/airflow/README.md:1]: We will still be publishing the `marquez-airflow` lib, so I wouldn't mark the module as deprecated. All the other changes to the `README.md` look good.

Comment (mobuchowski) [reply integrations/airflow/README.md]: Changed.

**Comment by mobuchowski** [comment]
@wslulciuc done.

**Review by wslulciuc — APPROVED** [review]
LGMT 👍

**Pull request merged**